resources/lang/pl_PL/tables.yml

```diff
@@ -28,6 +28,21 @@ tables:
       facility: "$t(models.member.facility)"
       hasFacilityAdmin: "$t(models.member.hasFacilityAdmin)"
       actions: "$t(tables.tables.generic.columnNames.actions)"
+  facilities:
+    tableName: "$t(models.facility._name_plural)"
+    summary__zero: "Brak placówek"
+    summary__one: "{{count, number}} placówka"
+    summary__few: "{{count, number}} placówki"
+    summary__many: "{{count, number}} placówek"
+    columnNames:
+      id: "$t(tables.tables.generic.columnNames.id)"
+      name: "$t(models.facility.name)"
+      createdAt: "$t(models.generic.createdAt)"
```

Comment:
Tu użyj models.facility, a nie models.generic (chociaż oczywiście models.facility może przekierowywać na models.generic). W ten sposób będzie można override'ować jakąś nazwę w modelu facility i się od razu zaktualizuje tabela.

Reply:
Ok, ale co ty na to, żeby tabela miała opcję, że szuka models.yml i dopiero jak nie znajdzie to w tables.yml? Bo po co mamy tak przekierowywać wszystkie pola. Mogę tak przerobić w oddzielnym PR.